website/docs/docs/deploy/job-notifications.md

```diff
@@ -56,17 +66,20 @@ You can receive email alerts about jobs by configuring the <Constant name="cloud
 1. Select your profile icon and click on **Notification settings**.
 1. On the **Email notifications** page, click **Unsubscribe from all email notifications**. 
 
-## Slack notifications
-
+## Slack notifications (user-linked) {#slack-notifications}
 You can receive Slack alerts about jobs by setting up the Slack integration and then configuring the <Constant name="cloud" /> Slack notification settings. <Constant name="cloud" /> integrates with Slack via OAuth to ensure secure authentication.
 
+This is the current Slack integration available for all users and not to be confused with the [Slack notifications at the account level](#slack-notifications-account) feature, which is available only in private beta. To request access, contact your account manager. Only refer to these instructions if you _don't_ have access to the private beta features.
```

Comment:
clarifying diff between user linked and account linked for current users so they're not confused with which one to use/follow